.github/workflows/wagon-tests.yml

```diff
@@ -41,6 +41,9 @@ defaults:
     working-directory: hitobito
 
 jobs:
+  detect_wagon_dependency:
+    # TODO
```

Comment:
Vorschlag:
- checkout/cd wagon-repository
- `grep "add_dependency.*hitobito_" *.gemspec | sed 's/.*hitobito_\([-_a-z]*\).*/\1/'`

Das gibt z.B. für die Jubla "youth" zurück. man kann das sed sicherlich noch anpassen, dass man direkt "hitobito_youth" zurückbekommt.

Reply:
Sounds good, I was thinking along the same lines. Maybe the git checkout could be sparse (only checkout the gemspec or lib dir) and only checkout the master branch.